salt/common/tools/sbin/so-status

```diff
@@ -290,22 +291,29 @@ main() {
 
 # {% endraw %}
 
+is_tty() {
+    __tty=0
+    [ -t 1 ] \
+        && __tty=1
+    # don't print colors if NO_COLOR is set to anything
+    [ "${#NO_COLOR}" -ne 0 ] \
+        && __tty=0
+    return "${__tty}"
```

Comment:
There's no need to return if you aren't doing anything with the return value.

Reply:
Fixed